pages/account/manage/link/[[...data]].js

```diff
@@ -60,12 +64,13 @@ export default function ManageLink({ BASE_URL, username, link }) {
   const [icon, setIcon] = useState(link.icon || "");
   const [isEnabled, setIsEnabled] = useState(link.isEnabled ? true : false);
   const [isPinned, setIsPinned] = useState(link.isPinned ? true : false);
-
+  const [animation, setAnimation] = useState(link.animation || "none");
```

Comment:
This should be from the config, maybe something like?

```suggestion
  const [animation, setAnimation] = useState(link.animation || Object.keys(config.animations)[0]);
```